assets/js/modules/analytics/components/common/UseSnippetSwitch.js

```diff
@@ -66,7 +83,7 @@ export default function UseSnippetSwitch() {
 				checked={ useSnippet }
 				onClick={ onChange }
 				hideLabel={ false }
-				disabled={ ! canUseSnippet }
+				disabled={ ! canUseSnippet || toggleSwitchDisabled }
```

Comment:
See above. It seems unnecessary to disable the snippet control here? In the setup we currently don't show it all right? So while we need to set `useSnippet` to `false` during setup if there's an existing tag, I'm not sure why it's necessary to disable the toggle. Can you clarify? Maybe I missed something from another conversation.